perma_web/perma/templates/registration/check_email.html

```diff
@@ -9,6 +9,11 @@
 			<h1>Thanks for creating an account</h1>
 			
 			<p>Look for an email from Perma and click on the link to activate your account.</p>
+			    {% if messages %}
+        {% for message in messages %}
+            <p>{% if 'safe' in message.tags %}{{ message|safe }}{% else %}{{ message }}{% endif %}</p>
+        {% endfor %}
+    {% endif %}
```

Comment:
Nitpick -- let's consistently indent blocks with 4 spaces in templates, the same way we do in Python. So, `{% if %}` would line up with `<p>`, and `{% endif %}` would line up with `{% if %}`. It's not crazy confusing here, but it can definitely lead to confusion and bugs as templates get edited.